test/check-terminal

```diff
@@ -37,11 +37,13 @@ class TestTerminal(MachineCase):
         def wait_line(i,t):
             b.wait_text(line_sel(i), line_text(t))
 
+        b.wait_present('#terminal .terminal')
         b.focus('#terminal .terminal')
 
         # wait for prompt in first line and remember it
         b.wait_text_not(line_sel(1), line_text(""))
-        prompt = b.text(line_sel(1))
+        prompt = u"[admin@m10\xa0~]$"
```

Comment:
the hostname is not necessarily m10, this causes tests to fail
m## or even broader matching would be better here, I think